### PR TITLE
Changes to structure

### PR DESCRIPTION
**Pull request by RomanSavran**
- All attributes added to ontology and mapped to domains
- Context and vocabulary files autogenerated
- Ontology file changed due to latest requerements
- Documentation file changed

### CODE REVIEW

File: vocabularies/person.jsonld [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                "dli:required": true
+            },
+            {
+                "@type": "pot:SupportedAttribute",
+                "dli:attribute": "dli:data",
+                "dli:title": "data",
+                "dli:description": "data",
+                "dli:required": true,
+                "dli:valueType": "xsd:object"
+            }
```
Comment (soderluk) [inline vocabularies/person.jsonld]: There should probably be some more information about a person. E.g. optional attributes such as date of birth etc. other useful information that a "person" can have.

File: vocabularies/link.jsonld [hunk 2]
```diff
@@ -1,32 +1,47 @@
...
+                "dli:required": true
+            },
+            {
+                "@type": "pot:SupportedAttribute",
+                "dli:attribute": "dli:data",
+                "dli:title": "data",
+                "dli:description": "data",
+                "dli:required": true,
+                "dli:valueType": "xsd:object"
+            }
```
Comment (soderluk) [inline vocabularies/link.jsonld]: Please revert this file, as links do only have a type. No name or data attribute at the moment.

File: README.md [hunk 3]
```diff
@@ -1,182 +1,63 @@
...
+Data model include real life objects and their properties.
+We have a set of classes, arranged in a multiple inheritance hierarchy where each class may be a sub-class of multiple classes.
+We also have a set of properties:
+* each property may have one or more classes as its domains. The property may be used for instances of any of these classes.
+* each property may have one or more classes as its ranges. The value(s) of the property should be instances of at least one of these classes.
+* can be arranged in hierarchy same as classes.
+
+The decision to allow multiple domains and ranges was purely pragmatic. While the computational properties of systems with a single domain and range are easier to understand, in practice, this forces the creation of a lot of artifical classes, which are there purely to act as the domain/range of some properties.
+
+All classes and properties can be investigated in [Data Model structure documentation](https://romansavran.github.io/PoT/).
```
Comment (soderluk) [inline README.md]: Please point to the standards repository instead.

File: README.md [hunk 4]
```diff
@@ -1,182 +1,63 @@
...
+* Core ontology file is named `pot.jsonld` and is located under `ontologies` folder. It contains all classes and properties used in ontology
+* Context files are located in `contexts` folder and contains one file per identity. This file describes JSON document sctructure of specific identity.
+* Vocab files are located in `vocabularies` folder and describes properties metadata like labels on forms and requirements
+
+Core ontology should be understood as core terms vocabulary including all classes and properties. Context files define JSON document strcuture and are connected to ontology. Vocab files should be used to build UI and other application level represetations of data structure and also to understand some technical level properties of attributes.
+
+All JSON-LD source files are available under [GitHub Repository](https://github.com/PlatformOfTrust/standards).
+
+## Links and additional information
+All classes and properties can be investigated in [Data Model structure documentation](https://romansavran.github.io/PoT/).
```
Comment (soderluk) [inline README.md]: Please point to the standards repository.

**Review by soderluk — CHANGES_REQUESTED** [review]
You have removed all the contexts for links, which we need in the system. Please restore ALL link-*.jsonld contexts, and fix the vocabularies for them.
Also, you removed one of the most important context file, e.g. identity-app.jsonld, please restore the identity-app.jsonld context.

**Review by soderluk — COMMENTED** [review]
The context file identity-data_product.jsonld is removed, please restore it.

**Review by soderluk — COMMENTED** [review]
The identity-link.jsonld shouldn't exist. A link is not an identity, it's a link.
Links do not have a name or data, only the type

**Comment by soderluk** [comment]
Could you at the same time update (replace) the base URL from `https://platformoftrust.github.io/standards/` to `https://standards.oftrust.net/`
Should be pretty straight forward to find and replace the previous string with the latter.

File: vocabularies/person.jsonld (hunk 1 above)
Comment (RomanSavran) [reply vocabularies/person.jsonld]: Redirected this to Sebastian - he is providing attributes list e.t.c.

**Comment by soderluk** [comment]
@RomanSavran: Please merge master into your branch, and resolve the conflicts.

**Comment by RomanSavran** [comment]
@soderluk I completed pot ontology part, but it seems like DLI ontology https://digitalliving.github.io/standards/ontologies/dli.jsonld is outdated (it is missing some classes like Building) even so this classes should be there instead of pot according to latest input I have

Is it possible to provide me access to https://digitalliving.github.io/standards/ so I can add missing classes or how would it better solve this?

**Comment by soderluk** [comment]
@RomanSavran Did you get the invite?

**Comment by RomanSavran** [comment]
@soderluk Got it, thank you

File: vocabularies/person.jsonld (hunk 1 above)
Comment (RomanSavran) [reply vocabularies/person.jsonld]: Will merge this and work with Sebastian on attributes on Person and other classes in next commits

**Pull request merged**